src/System.Net.Requests/src/project.json

```diff
@@ -8,6 +8,7 @@
         "System.Diagnostics.Tracing": "4.0.0",
         "System.IO": "4.0.10",
         "System.IO.Compression": "4.1.0",
+        "System.Net.Http": "4.0.0",
```

Comment:
Was this there already in 1.1?  Because I think it needs to be 4.1.0.  The 4.0.0 System.Net.Http dll is the one used in NETFX and it doesn't have the right properties such as ServerCertificateCustomValidationCallback on the HttpClientHandler.